src/components/20-molecules/top-content-bar/README.md

```diff
@@ -0,0 +1,75 @@
+# Top content bar
+
+Used as top of the page Warning or info box that can show text and have a CTA
+
+## Usage
+
+```bash
+npm install @axa-ch/top-content-bar
+```
+
+```js
+import '@axa-ch/top-content-bar';
+...
+<axa-top-content-bar ctatext="Click Me">Some Text</axa-top-content-bar>
+```
+
+### React
+
+Create a React-ified top-content-bar with the createElement function from your React version and then use it like this:
+
+```js
+import { createElement } from 'react';
+import createAXATopContentBarReact from '@axa-ch/top-content-bar/lib/index.react';
+
+const AXATopContentBarReact = createAXATopContentBarReact(createElement);
+
+export default AXATopContentBarReact;
+```
+
+```js
+<AXATopContentBarReact onClick={handler} ctatext="Click Me" />
```

Comment:
What is 'handler'? In other readmes we write down all properties. Not sure if we should, but we do it.

Reply:
generated from scaffolding and it just shows that there u can write a handler so its fine